### ISSUE TITLE
Cant't craft dirt from dirt resin and dirt acorn.

### ISSUE THREAD

**Issue by gebab-code** [Darkosto/SkyFactory4Issues#962]
## Bug Report

Placing 5 dirt resin and 4 dirt acorn in an 3x3 crafting grid, acording to recipe, can't take dirt out. It is not shown.

## Expected Behaviour

A dirt block is expected.

## Steps to Reproduce (for bugs)

1. Create a SF4 Server.
2. Join Server and try to craft dirt.

## Logs

* Client/Server Log:
* Crash Log:

## World Information

* Preset: Skyfactory 4

* Prestige: yes

* Modpack Version world created in: 4.0.5

* Additional Content Installed: I needed to disable withercrumbs, because it caused the server not to start.

## Client Information

* Modpack Version: 4.0.5
* Java Version: 1.8.0_51 64bit
* Launcher Used: Twitch
* Memory Allocated: 7GB
* Server/LAN/Single Player: Server (Only me / no other players)
* Optifine Installed: no
* Shaders Enabled: no

## Server Information
* Java Version: Java is OpenJDK 64-Bit Server VM, version 1.8.0_171, running on Linux:amd64:4.15.0-47-generic
* Operating System: Ubuntu 18.04.2 LTS
* Hoster/Hosting Solution: Self Hosted
* Sponge (Non-Vanilla Forge) Server: n/a

It works in singleplayer.

**Comment by sk2048** [comment]
Withercrumbs items are referenced in the crafttweaker scripts, removing it removes the item and causes the scripts to error. As a result you have no recipes.

**Comment by sk2048** [comment]
Withercrumbs is also required for some advancements to work properly. I suggest you try to work out why that mod is crashing on your server.

**Comment by gebab-code** [comment]
Ah, OK. Thanks. Will try to figure it out.

**Comment by sk2048** [comment]
You can join the discord and ask in #sf4-server-support to see if anyone else has had issues, or upload your crash report there and see if anyone can tell what's causing it.

**Comment by sk2048** [comment]
https://discord.gg/darkosto

**Comment by gebab-code** [comment]
Hello. I got it solved. Since WitherCrumb crashes with "Invalid JSON Object" or so, i looked at the CustomWithers.json file and it contained an emtpy JSON Array of []. I deleted it. and restarted the server.
The Server is now running, and the recipe works. Thanks

> Withercrumbs items are referenced in the crafttweaker scripts, removing it removes the item and causes the scripts to error. As a result you have no recipes.

Hello. I got it solved. Since WitherCrumb crashes with "Invalid JSON Object" or so, i looked at the CustomWithers.json file and it contained an emtpy JSON Array of []. I deleted it. and restarted the server.
The Server is now running, and the recipe works.
Thanks for pointing me in the right direction. I had no idea that withercrumbs has something to do with crafttweaker.
